xc/xenops_server_xen.ml

```diff
@@ -3329,31 +3329,52 @@ module Actions = struct
       )
     in ()
 
+  let xenbus_connected = Xenbus_utils.(int_of Connected) |> string_of_int
```

Comment:
if you were to put this inside the body of `maybe_update_pv_drivers_detected`, the compiler should optimise that

Reply:
I'm not certain it would!